nuxt.config.js

```diff
@@ -42,11 +44,22 @@ export default {
     // https://go.nuxtjs.dev/axios
     '@nuxtjs/axios',
   ],
-
+  publicRuntimeConfig:{
+    API_KEY:process.env.API_KEY,
+  },
   // Axios module configuration: https://go.nuxtjs.dev/config-axios
   axios: {
     // Workaround to avoid enforcing hard-coded localhost:3000: https://github.com/nuxt-community/axios-module/issues/308
-    baseURL: '/',
+    baseURL: 'https://newsapi.org/v2/',
+    Headers:[
+      {
+        "Access-Control-Allow-Origin":"*"
+      }
+    ]
```

Comment:
Create a plugin for axios configuration

Reply:
@10shubham01  I have already added plugin for axios, do you mean config file to add configuration in that file ?

Reply:
yes

Reply:
resloved